src/app/pass-lookup/pass-lookup.component.scss

```diff
@@ -0,0 +1,36 @@
+@import "/src/assets/base/base.scss";
+
+@import url(https://fonts.googleapis.com/css?family=Roboto:400,100,900);
+
+.back-button {
+    display: inline-flex;
+    vertical-align: middle;
+}
+
+.card-header {
+    background-color: #003366;
+    color: white;
+}
+
+.park-select-header {
+    background-color: #003366;
+    color: white;
+}
+
+.registration-details {
+    background-color: #38598a;
+    border-radius: 5px;
+    color: white;
+    margin: 20px;
+    padding: 20px;
+}
```

Comment:
Some of these should go into the base css class, and the margin/padding needs to be bootstrap style.

Reply:
moved to `base.scss`